README.md

```diff
@@ -1,75 +1,79 @@
-<a href="https://codeclimate.com/github/techly-africa/mpa-cash-backend/maintainability"><img src="https://api.codeclimate.com/v1/badges/1f6c2024dcbec25991dc/maintainability" /></a>
-<a href='https://coveralls.io/github/techly-africa/mpa-cash-backend?branch=main'><img src='https://coveralls.io/repos/github/techly-africa/mpa-cash-backend/badge.svg?branch=main' alt='Coverage Status' /></a>
-<p align="center">
-  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo-small.svg" width="200" alt="Nest Logo" /></a>
-</p>
+# Role-Based Access Control Backend(Auth-X)
```

Comment:
could you re add the coverage badge from coveralls?

Reply:
Yes @edwinnambaje 
The Badge is there but i think we need to Update it since the repo's name has changed

Reply:
If you have a new one please share it with me and i update it 
Thanks

Reply:
i will add it there no problem